package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "buttercup-browser-extension",
-  "version": "3.0.0",
+  "version": "3.1.1",
```

Comment:
No need to change the version. This is handled during release.

Reply:
Will drop it. However, I just want to point out that the source code is still at 3.0.0, even though v3.1.0 is in the wild.